src/lib.rs

```diff
@@ -9,12 +9,14 @@ mod ic0_memory; // Memory API for canisters.
 pub mod log;
 pub use log::{Log as StableLog, Log};
 pub mod memory_manager;
+pub mod min_heap;
 pub mod reader;
 pub mod storable;
 #[cfg(test)]
 mod tests;
 mod types;
 pub mod vec;
+pub use min_heap::MinHeap as StableMinHeap;
```

Comment:
nit: should we also export the name without the stable prefix, as we do with other types?

```suggestion
pub mod vec;
pub use min_heap::{MinHeap as StableMinHeap, MinHeap};
```

Reply:
Done.